contrib/luarc.json

```diff
@@ -13,6 +13,13 @@
           "teardown",
           "finally",
           "lfs"
+        ],
+        "disable": [
+          "luadoc-miss-see-name",
+          "duplicate-doc-field",
+          "undefined-doc-name",
+          "undefined-field",
+          "param-type-mismatch"
```

Comment:
`undefined-doc-name` is useful, why disable that? Are there many cases that can't be fixed? (Example?)

The idea was to disable `luadoc-miss-see-name`, I don't know what all these other settings do. How many unfixable warnings are there without disabling these others? Would be helpful to see examples of each case

Reply:
line  375 has `  --- @field timer nil|uv_timer_t uv_timer`  caused by `uv_timer_t` is luv type.

Reply:
And can't that be handled with an annotation/type alias? The first choice should _always_ be to improve the typing rather than just to silence warnings.

(And it _really_ helps if you keep PRs focused to a single task instead of adding (prima facie) unrelated changes.)

Reply:
@clason check new comment and commit . thanks i do some change :)